org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/IsLaunchableHTMLTester.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Andrew Obuchowicz (Red Hat Inc.) 
+ *******************************************************************************/
+package org.eclipse.wildwebdeveloper.debug;
+
+import org.eclipse.core.expressions.PropertyTester;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.Adapters;
+import org.eclipse.wildwebdeveloper.debug.firefox.FirefoxRunDebugLaunchShortcut;
+
+public class IsLaunchableHTMLTester extends PropertyTester {
+	private static final String PROPERTY_NAME = "isHTMLLaunchable"; //$NON-NLS-1$
+	@Override
+	public boolean test(Object receiver, String property, Object[] args, Object expectedValue) {
+		if (property.equals(PROPERTY_NAME)) {
+			IResource resource = Adapters.adapt(receiver, IResource.class);
+			if (resource == null) {
+				return false;
+			}
+			return FirefoxRunDebugLaunchShortcut.canLaunch(resource);
```

Comment:
`new FirefoxRunDebugLaunchShortcut().canLaunch(resource.getLocation().toFile())` would allow to reuse already existing code.